### PR TITLE
Fix changing one filter settings resets the other

### PR DESCRIPTION
**Pull request by cosa65**
# Background
#### Link to issue 
https://biomage.atlassian.net/browse/BIOMAGE-881

#### Link to staging deployment URL 
https://ui-martinfosco-ui234.scp-staging.biomage.net/

#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [x] Tested locally with Inframock (with latest production data downloaded with biomage experiment pull)
- [x] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

File: src/components/SliderWithInput.jsx [hunk 1]
```diff
@@ -14,11 +14,21 @@ const SliderWithInput = (props) => {
...
 
   const [localValue, setLocalValue] = useState(value);
+  const [writtenLocalValue, setWrittenLocalValue] = useState(value);
 
   useEffect(() => {
-    setLocalValue(value);
+    setLocalValue(parseFloat(value));
   }, [value]);
 
+  useEffect(() => {
```
Comment (cosa65) [inline src/components/SliderWithInput.jsx]: These changes were a minor change of behavior to have a delayed run of `handleChange` after writing numbers in the input box, unrelated to the main issue addressed in the ticket.

**Review by xverges — COMMENTED** [review]
@cosa65 Can we have some unit test added to src/__test__/redux/reducers/experimentSettingsReducer.test.js ?

**Review by xverges — COMMENTED** [review]
Code looks good. Will play a bit now before approving

**Comment by xverges** [comment]
@cosa65 Freshly restarted inframock, with an experiment downloaded a few days ago with `biomage experiment pull`. As soon as I go to `Data Processing` and switch to the `Cell size distribution filter` I get `Save Changes` triggered and a warning about changes applied to a single sample.
image

**Review by xverges — APPROVED** [review]

**Pull request merged**